README.md

```diff
@@ -8,6 +8,15 @@ A youtube downloader written in Javascript. To be used with the command line. If
 
 # Usage
 
+
+Download video and convert to mp3 (Requires ffmpeg)
+
+```bash
+ name=`echo myfilename` && ytdl http://www.youtube.com/watch?v=_HSylqgVYQI > $name.mp4 && ffmpeg -i $name.mp4 -b:a 192K -vn $name.mp3 && rm -rf $name.mp4
```

Comment:
Video can be piped directly into ffmpeg without having to save it to a file and then deleting it

    ytdl http://www.youtube.com/watch?v=_HSylqgVYQI | ffmpeg -i pipe:0 -b:a 192K -vn myfile.mp3

Reply:
Added the direct pipe example as you provided.